docs/self-hosted-appcircle/configure-server/monitoring.md

```diff
@@ -0,0 +1,186 @@
+:::
+
+The Grafana monitoring UI is accessible by default through the `monitor` subdomain. In our example, the full domain URL is `monitor.appcircle.spacetech.com`.
+
+:::caution
+You can face SSL errors while connecting to the `monitor` URL like `ERR_CERT_COMMON_NAME_INVALID`. That error indicates the SSL certificate of the Appcircle server doesn't include the `monitor` subdomain.
+
+The recommended solution is to update the SSL certificate of the Appcircle server.
+:::
+
+Upon navigating to the Grafana login page, you should authenticate using the initial username and password specified in the global.yaml file of your project. To verify these credentials, you can execute the following command on the Appcircle server:
+
+Change directory to Appcircle server.
+
+```bash
+cd appcircle-server
+```
+
+- Update the environment variable `PATH` with the required dependencies.
+
+```bash
+export PATH=$PATH:$(pwd)/deps/bin
+```
+
+Print the Keycloak authentication credentials.
+
+```bash
+yq '.keycloak.initialUsername' ./projects/spacetech/export/.global.yaml && \
+yq '.keycloak.initialPassword' ./projects/spacetech/export/.global.yaml
+```
+
+The example output below displays the `initialUsername` on the first line and `initialPassword` on the second line. These credentials serve as your login information for the Grafana monitoring interface.
+
+```text
+admin@spacetech.com
+SuperSecretPassword
+```
+
+## Retention Period of Logs
+
+Retention period refers to the duration for which log data is stored before being deleted or archived. It's used to manage storage space, optimize system performance, and ensure compliance with regulatory requirements.
+
+### Retention Period on Loki
+
+The Appcircle server logs will be stored in the Loki. So the queries and filters that you run from the Grafana UI will run on the Loki side.
+
+The logs in the loki should be cleaned automatically. By default, the retention period for the Appcircle server logs on the Loki side are 720 hours (30 days).
+
+If you want to change this, you can edit the `global.yaml` of your project.
+
+<DowntimeCaution />
+
+```bash
+vi ./projects/spacetech/global.yaml
+```
+
+Add or change the retention period variable.
+
+```yaml
+loki:
+  retentionPeriod: 168h # 7 days
+```
+
+<RestartAppcircleServer />
+
+### Retention on Systemd
+
+The Appcircle server and other system services also transmit their logs to the Journald log driver. However, once Appcircle server logs are successfully forwarded to Loki, the local server logs become redundant and can be safely deleted.
+
+If you wish to configure a maximum size limit for automatic log deletion on the Journald, you can modify the relevant configuration settings.
+
+:::info
+Modifying the Journald configuration requires elevated privileges with `sudo` permissions, as it involves altering system-level settings.
+:::
+
+Edit the Journald config file.
+
+```bash
+sudo vim /etc/systemd/journald.conf
```

Comment:
As a convention, we're using `vi` in sample commands. (not `vim`)

Reply:
Converted.